cmd/oci-runtime-tool/generate.go

```diff
@@ -22,8 +22,6 @@ var generateFlags = []cli.Flag{
 	cli.StringFlag{Name: "arch", Value: runtime.GOARCH, Usage: "architecture the container is created for"},
 	cli.StringSliceFlag{Name: "args", Usage: "command to run in the container"},
 	cli.StringSliceFlag{Name: "bind", Usage: "bind mount directories src:dest[:options...]"},
-	cli.StringSliceFlag{Name: "cap-add", Usage: "add Linux capabilities"},
-	cli.StringSliceFlag{Name: "cap-drop", Usage: "drop Linux capabilities"},
```

Comment:
Because the structure of  `capabilities` changes, I do not know how to deal with these two functions, so temporarily delete, there are good ways to solve.